clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -131,6 +131,7 @@ public RequestFuture<ClientResponse> send(Node node,
         ClientRequest clientRequest = client.newClientRequest(node.idString(), requestBuilder, now, true,
             requestTimeoutMs, completionHandler);
         unsent.put(node, clientRequest);
+        log.info("Generated FETCH with correlation id {}.", clientRequest.correlationId());
```

Comment:
Logging every request at INFO level, does sound very chatty. I would believe that per-request logging would even be too chatty for DEBUG, and we should make it TRACE instead?